src/App.js

```diff
@@ -22,10 +33,12 @@ class App extends Component {
         </Route>
         <Route path="/mysongs">
           <p>Made it to mysongs</p>
+          <Library songs={ this.state.mySongs }/>
```

Comment:
Do we maybe wanna just call this component SongBook?  or SongLibrary? to be more explicit?

Reply:
I could get behind that